### PR TITLE
Accessibility tweaks and additions

### PR DESCRIPTION
**Pull request by guermanc**
... including examples and resources.

### CODE REVIEW

File: apps/docs/src/common/pages/accessibility/keyboard/skip-to-content.tsx [hunk 1]
```diff
@@ -111,19 +111,13 @@ const Page: FC<PageProps> = ({ location }) => (
       <p>For keyboard users, having to repeatedly move through the same navigation menu every time they start a new page can be frustrating.</p>
       <p>This is why ‘skip to content’ links exist. The first tab move you make on each page should show a link at the top of the page that says ‘Skip to content’ (or similar) and moves the user to the main part of the page.</p>
 
-      <div className="govuk-grid-row">
-      <div className="govuk-grid-column-one-quarter">
-        <img src={skipImage01} alt="Screenshot of skip to main content link selected" className="image-examples" />
+      <div className="example" style={{marginBottom: "2em"}}>
+        <p>To view the skip link component tab to this example, or click inside this example and press tab.</p>
+        <A href="#" class="govuk-skip-link">Skip to main content</A>
```
Comment (daniel-ac-martin) [inline apps/docs/src/common/pages/accessibility/keyboard/skip-to-content.tsx:116]: `class=` needs to be changed to `className=`.

**Review by MarionDownie — COMMENTED** [review]
Audio and video - add a full stop at the end of the link.

Colour and contrast - repeats what’s at the bottom of this page: https://design.homeoffice.gov.uk/styles/colour Link instead.  
Full stop should also be outside of the link.

Images and timeouts - again repeat images from related pages in the design system - suggest linking instead.

Loving the skip to content example!

Resources - 'Easy Checks – A First Review of Web Accessibility'. Use sentence case (even though they haven't)

Tables - given the same example of 'Ilya, senior staff member who is blind' a few times - is it relevant to all?

**Comment by guermanc** [comment]
The images and examples in colour and contrast, images and timeouts are meant to illustrate and give some context to the guidance, which is why I've included these inline. 

I have trimmed down the colour combinations but kept the image and timeout examples as they're useful to have at a glance but let me know if you still want me to change these.

Removed Ilya, senior staff member who is blind from tables and headings, and kept it in the first page structure page, links, as well as images.

**Comment by MarionDownie** [comment]
Ok, I'll accept those changes. Could you please just add a link on the colour and contrast page through to the colour page for the full list of our colours? I want link pages across sections where possible.

**Comment by MarionDownie** [comment]
I think it's fine to push, though something about the intro doesn't feel right. I think I'd move the original second sentence back up to above the image ('It is important contrast is clear, though there also should be alternatives for those who cannot see clearly.') as it's more introductory.

**Review by MarionDownie — APPROVED** [review]

**Pull request merged**